src/mercury_engine_data_structures/formats/bmsld.py

```diff
@@ -55,10 +55,9 @@
     x=Float,
     y=Float,
     z=Float,
-    unk05=Hex(Int32ul),
-    unk06=Hex(Int32ul),
-    unk07=Hex(Int32ul),
-
+    x_rotation=Float32l,
+    y_rotation=Float32l,
+    z_rotation=Float32l,
```

Comment:
I know this will be a breaking change, but you should do it now rather than later: 
```py
position=CVector3D,
rotation=CVector3D,
```

Reply:
Yeah, I asked if I should do this for both in sr-dev channel but it was pushed away due to dcr and me talking about other things :D

Reply:
Done...and sucessfully broke OSRR and the SR editor